README.md

```diff
@@ -1,45 +1,57 @@
 # Interview Assignment - Coop Logistics API
 
-Before jump in:
+> This repository hosts the source code for a client designed to generate a
+> specified number of requests (N).
+>
+> As part of the assignment, the candidate is
+> required to develop an API server capable of handling these requests.
+> The server can be implemented using either gRPC or HTTP protocols.
+> 
+> The primary objective of this test is to assess the candidate's ability to
+> create a robust API server that efficiently processes incoming requests as
+> specified by the client's operations
```

Comment:
Using Markdown quotes combined with how GitHub renders them does not bring the attention you intended to this information. I would use a normal paragraph.

```suggestion
This repository hosts the source code for a client designed to generate a
specified number of requests (N).

As part of the assignment, the candidate is
required to develop an API server capable of handling these requests.
The server can be implemented using either gRPC or HTTP protocols.
 
The primary objective of this test is to assess the candidate's ability to
create a robust API server that efficiently processes incoming requests as
specified by the client's operations
```